.github/workflows/build_source_release.yml

```diff
@@ -0,0 +1,39 @@
+name: Build Source Release
+
+# Trigger whenever a release is created
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+
+    - name: archive
+      id: archive
+      run: |
+        VERSION=${{ github.event.release.tag_name }}
+        PKGNAME="SoftEtherVPN-$VERSION"
```

Comment:
I was considering renaming this to `SoftEther_VPN-$VERSION`, but the attached name matches the name of the repository.